docs/clipboard-unsanitized/explainer.md

```diff
@@ -0,0 +1,257 @@
+# Unsanitized HTML for Async Clipboard API
+
+
+## Author:
+*   ansollan@microsoft.com
+*   snianu@microsoft.com
+
+## Introduction
+Using DataTransfer object’s setData and async clipboard write method, we are seeing interop differences in how the HTML content is sanitized and written to the clipboard. In Chromium async clipboard write method clears the clipboard content first and then writes the payload which results in overwriting the previous HTML content that was inserted by authors using DataTransfer object’s setData API. It’d be beneficial for the web authors if async clipboard and setData APIs provide the same HTML content during copy operation so round tripping is possible without any interop differences.
```

Comment:
Not sure it makes sense to specifically talk about the Chromium implementation of the async clipboard write API. We can talk about how the behavior is spec'ed or how the behavior differs between browsers, or it might even be fine to exclude that part. I think the introduction can be insightful enough without the implementation details.

Reply:
I can add a note here if that would be helpful, but I'm not sure if I know enough to talk about the implementation of other browsers. I'll remove the Chromium implementation details.